go/packages/packages_linux.go

```diff
@@ -498,15 +497,15 @@ func installedGEM() ([]PkgInfo, error) {
 	lines := strings.Split(strings.TrimSpace(string(out)), "\n")
 
 	if len(lines) == 0 {
-		logger.Info("No gems installed.")
+		fmt.Println("No gems installed.")
 		return nil, nil
 	}
 
 	var pkgs []PkgInfo
 	for _, ln := range lines[2:] {
 		pkg := strings.Fields(ln)
 		if len(pkg) < 2 {
-			logger.Errorf("%q does not represent a gem", ln)
+			fmt.Printf("%q does not represent a gem\n", ln)
```

Comment:
Might be a good time to add the beginnings of a custom logger even if the logger just calls `fmt.Printf()`

Reply:
Did this for the main binary, these libraries still need to have something passed in which was a bit more than I wanted to setup now